cirq/linalg/states.py

```diff
@@ -21,6 +21,7 @@
 def one_hot(*,
             index: Union[None, int, Sequence[int]] = None,
             shape: Union[int, Sequence[int]],
+            value: int = 1,
             dtype: Type[np.number]) -> np.ndarray:
     """Returns a numpy array with a single 1 entry, and 0 everywhere else.
```

Comment:
update this top level doc string as it isn't correct now.

Reply:
Let me check

Reply:
@dabacon can you please elaborate more about this. I didn't understand. Which docstring you are talking about.

Reply:
Instead of `Returns a numpy array with a single 1 entry, and 0 everywhere else.` this should be something like
`Returns a numpy array with a single non-zero entry (default 1), and 0 everywhere else.`

Reply:
Thanks @dabacon please review it again.

Reply:
Ah sorry the string I gave you was too long so our pylint check is failing, maybe this is less than 80 chars.
`Returns a numpy array with all 0s and a single non-zero entry (default 1).`

Reply:
Thanks @dabacon I corrected doc string now.